### DISCUSSION TITLE
How can i publish dashboard.blade.php for customize ?

### DISCUSSION THREAD

**Discussion by farshidpg** [beyondcode/laravel-websockets#925] [Q&A] [answered]
hi how can i publish dashboard.blade.php for customize ?

**Comment by fredsal** [comment]
You can't

**Comment by erikn69** [comment, marked as answer]
Overwriting service provider
https://github.com/beyondcode/laravel-websockets/blob/fd2070fc3d28696247acceacc4f40667828f3016/src/WebSocketsServiceProvider.php#L105-L111
